tests/dummy/app/templates/application.hbs

```diff
@@ -6,4 +6,9 @@
   </div>
 {{/code-block}}
 
+<h3>With `line-numbers` plugin:</h3>
+{{#code-block class="line-numbers"}}&lt;div&gt;
```

Comment:
The `plugins` attribute could go without commas, but I hear what you're saying. Maybe I should move the code style to just be a class too then?